### DISCUSSION TITLE
Space Taxi: Speech

### DISCUSSION THREAD

**Discussion by mickeyl** [dirkwhoffmann/virtualc64#753] [General]
First off, congratulations for a great emulator on macOS. Running this on an arm-based Mac Studio.
I wonder about the lack of speech in Space Taxi. Is this a configuration issue or an incompatibility due to the SID emulation?

**Comment by dirkwhoffmann** [comment]
Most likely, you've selected a MOS 8580. Please try again with the MOS 6581.

**Reply by mickeyl** [reply]
Thanks, but that didn't change anything.

**Reply by mpkincai** [reply]
I confirmed that this didn't work on my copy of space taxi either.  Other sound effects work as expected.  I ran this in NTSC mode on an M1 Pro.

**Comment by dirkwhoffmann** [comment]
I've double checked with v4.6b1 and it works here. Please check if you have Audio Filters enabled (in addition to selecting the 6581):

<img width="909" alt="Bildschirm­foto 2023-01-27 um 19 54 46" src="https://user-images.githubusercontent.com/12561945/215171471-908610df-4ae9-4189-8350-29f3255b9269.png">

**Reply by neleabels** [reply]
I have verified "Space Taxi" with the settings "out of the box" (version 4.5.1, 2020 Macbook Pro) and the speech works nicely.
<img width="1280" alt="Bildschirm­foto 2023-01-28 um 08 38 52" src="https://user-images.githubusercontent.com/123799276/215253684-fe23e7b0-c937-4988-b59e-b5e05d01dbd2.png">

**Reply by mpkincai** [reply]
Thanks, this is working for me with the audio filter enabled.

**Reply by mickeyl** [reply]
Audio Filter on does it for me as well. Any particular reason why this isn‘t the default?
